docs/source/install/system_requirements.rst

```diff
@@ -28,9 +28,6 @@ be automatically added when you install |st2|.
 | `RHEL 7 <https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux>`_ /                 | `bento/centos-7.4 <https://app.vagrantup.com/bento/boxes/centos-7.4>`_       | `Red Hat Enterprise Linux (RHEL) 7.2 (HVM)  <https://aws.amazon.com/marketplace/pp/B019NS7T5I/ref=srh_res_product_title?ie=UTF8&sr=0-2&qid=1457037671547>`_       |
```

Comment:
On line 25 above, can you update for RHEL/CentOS 8.2?

Reply:
I've updated 8.2 centos. The only 8.2 AWS EC2 I could find is one from pro-computers. The AWS marketplace one is still the one above. Which do we want to link to?